timed/tracking/views.py

```diff
@@ -267,8 +267,7 @@ def bulk(self, request):
         if "task" in fields:
             # unreject report if task has changed
             fields["rejected"] = False
-            if fields["task"].project.billed:
-                fields["billed"] = fields["task"].project.billed
+            fields["billed"] = fields["task"].project.billed
```

Comment:
This could set `fields["billed"]` to `None` which is not the same as `False`. Not sure if it would be appropriate to default `fields["billed"]` to `False`, similar to the above `fields["rejected"]`

Reply:
i make it sure that the value will be true or false